tests/test_packages/unit/test_source_suppress.py

```diff
@@ -78,7 +78,7 @@ def test_multi_liner_all_2(self):
             sp_handler.suppress_comment(), 'some really long comment')
 
     def test_one_liner_some_checkers(self):
-        """Doc."""
+        """There is suppress comment above the bug line."""

```

Comment:
A more specific comment would be better like:
'Tests if there is a one liner suppress comment with multiple checkers in it'